proposals/20210525-examplefilter.md

```diff
@@ -0,0 +1,74 @@
+#### SIG TFX-Addons
+# Project Proposal
+
+**Your name:** Ryan Clough
+
+**Your email:** rclough@spotify.com
+
+**Your company/organization:** Spotify
+
+**Project name:** Example Filter
+
+## Project Description
+Beam based component that can filter Examples based on a user-defined predicate function.
+
+## Project Category
+Choose 1: Component
+
+## Project Use-Case(s)
+Data can be imported into TFX in a number of ways, and indeed, sometimes the dataset you wish to load is not under your direct 
+control. In cases like these, it is useful to have a component that can filter your input data with simple rules. Ex: filter 
+all records where `feature_a >= 1`. 
+
+Our organization currently has a component for this purpose that is in active use. It is not as robust as it could be.
+
+It is also worth conidering that we may wish to try and promote this functionality to be included in the TFX core base ExampleGen,
+so that the filtering could be done within any ExampleGen based component.
```

Comment:
Depending on whether this direction is pursued, would it result in different user facing API?